mozo.in

```diff
@@ -28,7 +28,7 @@ def main():
         version = config.VERSION
     except:
         datadir = './data/'
-        version = '1.7.0'
+        version = '1.21.0'
```

Comment:
This is weird... no, not your changes. I mean, why do we have some fallback here? If we can't import `config`, the package is damaged, and we should just quit.